test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Note that previous new files related with #2193 went to cases/0787... directory and used _in_ngsIv2.test suffix.

Not sure what is the best naming/ubication convention.